# MarginalGainRAG.msg
# bool flag_completed_action_selection: flag to indicate whether the publisher drone has finished action selection in its RAG round
# float64 best_marginal_gain: the best marginal gain corresponding to the optimal control action for the publisher drone
Header header
bool flag_completed_action_selection
float64 best_marginal_gain